tools/shared/src/main/scala/org/scalajs/core/tools/classpath/builder/JarTraverser.scala

```diff
@@ -61,9 +61,10 @@ trait JarTraverser extends ClasspathContentHandler with FileSystem {
     def entryBinaryContent: Array[Byte] = IO.readInputStreamToByteArray(in)
     def entryVersion: Option[String] = Some(entry.getTime.toString)
 
-    if (longName == JSDependencyManifest.ManifestFileName)
-      handleDepManifest(JSDependencyManifest.read(entryReader))
-    else if (longName.endsWith(".js")) {
+    if (longName == JSDependencyManifest.ManifestFileName) {
+      lazy val vf = JSDependencyManifest.read(entryReader)
+      handleDepManifest(vf)
```

Comment:
Why this change? Do you just think the line is too long? I think in any case you should make this a def. We don't need the lazy behavior.